environment_set_up/README.md

```diff
@@ -1,22 +1,32 @@
+3) Your own JupyterHub Instance<br>
+   Your can also work with your own deployment of a JupyterHub environment. You will need to make sure you have a kernel with all of the required packages set up to run these notebooks.
+
+
 # HPC Environment
 
-While these tutorial notebooks are intended for use on "_cloud_" infrastructure, it is possible to run them
-on HPC hardware also.  Some notebooks will need modifications specific to the compute cluster hardware.
-More importantly, you will also need to access an enviroment which emulates the Jupyter server -- where the
-notebooks will reside and execute -- using the HPC hardware. There are many ways to do this. Here are three
-options, in increasing order of complexity and flexibility:
+While we prefer that these notebooks be run on "_cloud_" infrastructure, we also provide some guidance for running them on USGS's supercomputers. You will need to access an enviroment which emulates the Jupyter server -- where the notebooks will reside and execute -- using the HPC hardware. There are many ways to do this. Here are three options, in increasing order of complexity and flexibility:
 
 1) [Open OnDemand](./OpenOnDemand.md)<br>
-   Provides the most effortless access to HPC hardware using a web interface (not unlike the
-   ESIP/QHUB interface). Only runs on `tallgrass`.
+   This option provides the most effortless access to HPC hardware using a web interface. Only runs on the [`Tallgrass`](https://hpcportal.cr.usgs.gov/hpc-user-docs/supercomputers/tallgrass.html) supercomputer.
 2) [Jupyter Forward](./JupyterForward.md)<br>
    This option gives you more control over how to launch the server, and on which host (can be
-   run on `denali`, `tallgrass`, or other hosts to which you have a login).  Requires that you
-   install an extra bit of software on your PC.
+   run on [`Denali`](https://hpcportal.cr.usgs.gov/hpc-user-docs/supercomputers/denali.html), [`Tallgrass`](https://hpcportal.cr.usgs.gov/hpc-user-docs/supercomputers/tallgrass.html), or other hosts to which you have a login).  Requires that you
+   install the jupyter-forward software on your PC.
 3) [Custom Server Script](./StartScript.md)<br>
    This option lets you completely customize your HPC compute environment and invoke the Jupyter
    server from a command shell on the HPC. Requires familiarity with the HPC command line, file
    editing, etc.
 
-If your workflow makes use of parallelism using Dask on HPC hosts, read [this](./tallgrass_dask-jobqueue.md)
-for information about how to configure the Slurm job scheduler within your notebook.
\ No newline at end of file
+If your workflow makes use of parallelism using Dask on HPC hosts, you will need to read [this](./tallgrass_dask-jobqueue.md)
```

Comment:
This looks like it is only for Tallgrass - do we have a similar set of instructions for Denali?

Also - we discussed deleting this file because it is identical to this notebook: https://github.com/hytest-org/hytest/blob/main/essential_reading/ParallelProcessing/Parallel_DaskJobQueue.ipynb. Is this true? If this is required for set-up, we should probably move this into this environment_set_up folder.